docs/cloud-workload-security/linux_expressions.md

```diff
@@ -2771,6 +2774,27 @@ Constants: [Open flags](#open-flags)
 
 
 
+### `open.syscall.flags` {#open-syscall-flags-doc}
+Type: int
+
+Definition: flags argument of the syscall
+
+
+
+### `open.syscall.mode` {#open-syscall-mode-doc}
+Type: int
+
+Definition: mode argument of the syscall
+
+
+
+### `open.syscall.path` {#open-syscall-path-doc}
+Type: string
+
+Definition: path argument of the syscall
```

Comment:
Optional: It's not consistent, but majority of the definitions seems to start capitalized

```suggestion
### `open.syscall.flags` {#open-syscall-flags-doc}
Type: int

Definition: Flags argument of the syscall



### `open.syscall.mode` {#open-syscall-mode-doc}
Type: int

Definition: Mode argument of the syscall



### `open.syscall.path` {#open-syscall-path-doc}
Type: string

Definition: Path argument of the syscall
```